05/Validator.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+class Validator {
+
+    public static function string($value, $min = 1, $max = INF) {
+        $value = trim($value);
+
+        return strlen($value) >= $min && strlen($value) <= $max;
```

Comment:
Pamiętaj, że w ten sposób liczymy bajty, a nie znaki. Z polskimi znakami to nie zadziała. Lepiej użyć `mb_strlen`